.github/workflows/mapi.yml

```diff
@@ -43,11 +43,10 @@ jobs:
 
     # Run Mayhem for API
     - name: Run Mayhem for API to check for vulnerabilities
-      uses: ForAllSecure/mapi-action@v1
+      uses: ForAllSecure/mapi-action@v2
       continue-on-error: true
       with:
-        mapi-token: ${{ secrets.MAPI_TOKEN }}
-        mapi-url: ${{ secrets.MAYHEM_URL }}
```

Comment:
I recommend keeping this. But rather than a secret, just make it a plain string. ie:

```
mapi-url: https://app.mayhem.security
```

This provides a clue where the url is overidable for enterprise deploys at different subdomains (ie: `https://customer.mayhem.security`)

Reply:
Added `mayhem-url: https://app.mayhem.security` in c870783746a1b09c2e8c9e02849dd79a96638263.

Reply:
Also added `mayhem-url: https://app.mayhem.security` in snippets in `README.md`.